pkg/kv/kvserver/replica_raft.go

```diff
@@ -380,7 +379,7 @@ func (r *Replica) propose(
 		log.Infof(p.ctx, "proposing command %x: %s", p.idKey, p.Request.Summary())
 	}
 
-	data, err := raftCmdToPayload(ctx, p.command, p.idKey)
+	data, err := raftlog.RaftCmdToPayload(ctx, p.command, p.idKey)
```

Comment:
The name is redundant. Consider `raftlog.CommandToPayload` or `raftlog.EncodeCommand`.